remote_jar_launcher/src/main/java/com/roche/sequencing/bioinformatics/common/java/JvmUtil.java

```diff
@@ -14,6 +14,7 @@
 import java.util.Comparator;

```

Comment:
What is your level of confidence that this will work with other users of this common utility like pt/peptide CLI?

Reply:
Or is this utility only referenced from AutoTestPlanCli?

Reply:
JvmUtil is currently only used for SlideViewer and PlatformTools (which is peptide CLI correct?).  It is not used by AutoTestPlanCli.    Please note that it will need to be updated with Java 9 since Oracle implemented a new versioning scheme.  This class is basically the runPlatformTools.jar which utilized the corresponding .runPlatformTools.cfg file.

Reply:
The only change in this class is related to the casting of a variable to double.  So the change is not related to HSQUtils.

Reply:
Yes - PlatformTools is what I am talking about for pt/peptide.
We'll need to coordinate when these systems start requiring Java 9, because the users do not have that JVM available in a lot of cases.  We can control that with deciding when to sync this with the SVN clone, and when to do builds of those apps, or with when to update the runPlatformTools.jar build.
I believe JVMUtil in bioinformatics.common.utils is used by Auto Test Plan (at least in the SVN/trunk version I am looking at) but this one in bioinformatics.common.java is in a different package.